pkg/cloudprovider/aws/apis/v1alpha1/provider_validation.go

```diff
@@ -163,3 +169,25 @@ func (a *AWS) validateStringEnum(value, field string, validValues []string) *api
 	}
 	return apis.ErrInvalidValue(fmt.Sprintf("%s not in %v", value, strings.Join(validValues, ", ")), field)
 }
+
+func (a *AWS) validateRootVolumeOptions() *apis.FieldError {
+	if a.RootVolumeOptions == nil {
+		return nil
+	}
+	if a.RootVolumeOptions.Size == nil {
+		return apis.ErrMissingField("rootVolumeOptions.size")
+	}
+	// 64 TiB is the maximum supported EBS volume size
+	maxRootVolumeSize := *resource.NewScaledQuantity(64, resource.Tera)
```

Comment:
Thoughts on putting these at the top as `var`

Reply:
I actually think it's not even worth tracking this min/max because it varies per volume type and might move at some point(?). I left it off in the next revision, let me know if you think we should bring it back.